packet/src/main/java/org/jclouds/packet/handlers/PacketErrorHandler.java

```diff
@@ -43,8 +43,6 @@ public void handleError(HttpCommand command, HttpResponse response) {
       message = message != null ? message : String.format("%s -> %s", command.getCurrentRequest().getRequestLine(),
               response.getStatusLine());
       switch (response.getStatusCode()) {
-         case 400:
-            break;
```

Comment:
If we don't handle this, we will propagate a generic `HttpResponseException` for bad requests. The common pattern is to handle it and encapsulate it in an `IllealArgumentException`.